### PR TITLE
Document HTTP compression feature

### PR DESCRIPTION
**Pull request by tanmayv25**

### CODE REVIEW

**Review by deadeyegoodwin — APPROVED** [review]

File: docs/inference_protocols.md [hunk 1]
```diff
@@ -47,6 +47,13 @@ model health, metadata and statistics. Additional endpoints allow
 model loading and unloading, and inferencing. See the KFServing and
 extension documentation for details.
 
+### HTTP Options
+Triton provides following configuring options for server-client network transactions over HTTP protocol.
```
Comment (CoderHam) [inline docs/inference_protocols.md:51]: the following configuration options

File: docs/inference_protocols.md [hunk 2]
```diff
@@ -70,7 +77,7 @@ Triton allows the on-wire compression of request/response messages by exposing f
 
 * `--grpc-infer-response-compression-level`
 
-For client-side documentation, see [Client-Side GRPC Compression](https://github.com/triton-inference-server/client/tree/main#compression)
+For client-side documentation, see [Client-Side GRPC Compression](https://github.com/triton-inference-server/client/tree/main#compression-1)
```
Comment (CoderHam) [inline docs/inference_protocols.md:80]: Why are you adding the -1 in the end?

**Review by CoderHam — CHANGES_REQUESTED** [review]

File: docs/inference_protocols.md (hunk 2 above)
Comment (tanmayv25) [reply docs/inference_protocols.md:80]: The compression without -1 links to http compression and compression-1 links to the second compression sub-heading for grpc.
https://github.com/triton-inference-server/client/tree/main#compression
https://github.com/triton-inference-server/client/tree/main#compression-1

I too was not aware of this way of indexing.

**Review by deadeyegoodwin — APPROVED** [review]

File: docs/inference_protocols.md (hunk 2 above)
Comment (CoderHam) [reply docs/inference_protocols.md:80]: TIL

**Review by CoderHam — APPROVED** [review]

**Pull request merged**